sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/samples/CreateNewSamplePage.java

```diff
@@ -78,6 +78,15 @@
   public static final By PATHOGEN_LABORATORY_INPUT =
       By.cssSelector(
           "[location='lab'] [class='v-filterselect v-widget v-required v-filterselect-required v-has-width'] input");
+  public static final By REQUEST_PATHOGEN_OPTION_BUTTON =
+      By.cssSelector("[id='pathogenTestingRequested'] label");
+  public static final By ANTIGEN_DETECTION_TEST_OPTION_BUTTON =
+      By.xpath("//label[text()='Antibody detection']");
```

Comment:
any other way to take the locator without visible text to make it reusable also for DE?

Reply:
I talk about this with team and we claim that option is the most safety and optional for this case. For DE case we simply create in future a new locators